privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -654,7 +654,7 @@ def testconnection(cls, param):
         
         return success, desc
 
-    def add_user(self, uid, object_class=None, attributes=None):
+    def add_user(self, uid, object_class=None, attributes=None, dn=None):
```

Comment:
No problem, Thank you.
Tried this out on my test LDAP server last night it works great.

However I did notice that maybe we could improve it further, this would also address one of
the issues that the callback functionality was envisioned achieve.
That being the addition of other LDAP attributes not mapped to PrivacyIDEA attributes,
my usage case being to have a LDAP attribute that marks a users as enabled or disabled, which could be used in the resolver ldap search.
The other improvement being the ability to select the resolver when adding a new user as there may be multiple for a single realm.

I'm more than happy to provide some code, if you think that these features may be useful.

Best Regards

Martin